app/views/posts/show.html.erb

```diff
@@ -3,14 +3,26 @@
   <h1><%= @post.text %></h1>
   <p><%= @post.created_at %></p>
   <p>Likes: <%= @post.likes.length %> </p>
+  <%= form_with model: like, method: :post, url: post_likes_path(@post, @user), local: true, scope: :new_like do |form| %>
+    <%= form.submit "Like", class: "btn btn-primary" %>
+    <%= form.hidden_field :post_id, value: @post.id %>
+    <%= form.hidden_field :author_id, value: @user.id %>
+  <% end %>
   <section>
     <h2>Comments</h2>
       <% @post.comments.each do |comment| %>
           <p><%= comment.text %></p>
           <p><%= comment.created_at %></p>
       <% end %>
   </section>
-
-
+  <section>
+    <h2>ADD a comment</h2>
+      <%= form_with model: comment, local: true, url:  post_comments_path(@post), scope: @comment, class: "my-form" do |form| %>
+        <%= form.text_area :text, placeholder: "Write your comment here", class: "my-text-area" %>
+        <%= form.hidden_field :post_id, value: @post.id %>
+        <%= form.hidden_field :author_id, value: @user.id %>
+      <%= form.submit class: "my-submit-button"%>
+  <% end %>
+  </section>
```

Comment:
- [ ] You have a nice approach of having the comment form on the `post#show` page, however, the project requirement is that the comment should have it's own view and this form should be rendered in the `views/comments/new.html.erb` file
Please implement the following routes for the comments 👇 
```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```